src/modules/appEvents.js

```diff
@@ -0,0 +1,30 @@
+const subscribers = [];
+/**
+ * Subscribe a callback to specified event.
+ *
+ * @param {string} eventName Name of the app event.
+ * @param {function} subscriberCallback Callback for the event.
+ */
+export function subscribeEventReceived(eventName, subscriberCallback) {
+  if (!Array.isArray(subscribers[eventName])) {
+    subscribers[eventName] = [];
+  }
+
+  subscribers[eventName].push(subscriberCallback);
+}
+
+/**
+ * Notify all subscribed callback when an app event is received.
+ *
+ * @param {stirng} eventName Name of the app event.
+ * @param {function} eventArguments Arguments of the event.
```

Comment:
The type is Array - at least its getting called from src/modules/appEvents.js with an Array